utils/hono-influxdb-connector/src/main/kubernetes/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright Bosch Software Innovations GmbH [and others], 2019. Part of the Eclipse Kuksa Project.
```

Comment:
I thought the deployment files go to `deploy/kubernetes/hono-influxdb-connector` not with the source code